cleaners/firefox.xml

```diff
@@ -18,51 +18,133 @@
-    <value search="glob" os="openbsd">~/.mozilla/firefox/*</value>
+    <value search="glob" os="windows">$$base$$\Profiles\*</value>
+    <value search="glob" os="linux">~/.$$Browser$$/*</value>
+    <value search="glob" os="freebsd">~/.$$Browser$$/*</value>
+    <value search="glob" os="openbsd">~/.$$Browser$$/*</value>
+  </var>
+  <var name="localprofile">
+    <value search="glob" os="windows">%LocalAppData%\$$Browser$$\Profiles\*</value>
+    <!-- Unknow by other OSes, yet! Tobias B. Besemer will do Linux Part, later! -->
+    <!--
+    <value search="glob" os="linux">~/.$$Browser$$/*</value>
+    <value search="glob" os="freebsd">~/.$$Browser$$/*</value>
+    <value search="glob" os="openbsd">~/.$$Browser$$/*</value>
+    -->
   </var>
   <option id="backup">
     <label>Backup files</label>
     <description>Delete the backup files</description>
+    <!-- Why not delete them all?
     <action command="delete" search="glob" path="$$profile$$/bookmarkbackups/*.json"/>
     <action command="delete" search="glob" path="$$profile$$/bookmarkbackups/*.jsonlz4"/>
+    -->
+    <action command="delete" search="walk.files" path="$$profile$$/bookmarkbackups/"/>
+    <!-- Done! -->
+    <action command="delete" search="walk.files" path="$$profile$$\sessionstore-backups\"/>
+    <action command="delete" search="file" path="$$profile$$\sessionstore-backups\"/>
+    <action command="delete" search="glob" path="$$profile$$/sessionstore.bak-*"/>
+    <action command="delete" search="glob" path="$$profile$$\*.bak"/>
+    <action command="delete" search="glob" path="$$profile$$\*.bak-*"/>
+    <action command="delete" search="glob" path="$$profile$$\prefs-*.js"/>
   </option>
   <option id="cache">
     <label>Cache</label>
     <description>Delete the cache</description>
     <!-- 
-This Linux path is whitelisted under the System - Cache cleaner, so it is not
-redundant. Also, this Linux path contains the cache2 and OfflineCache directories.
--->
-    <action command="delete" search="walk.all" path="~/.cache/mozilla/"/>
-    <action command="delete" search="walk.all" path="%LocalAppData%\Mozilla\Firefox\Profiles\*\cache2"/>
-    <action command="delete" search="walk.all" path="%LocalAppData%\Mozilla\Firefox\Profiles\*\jumpListCache"/>
-    <action command="delete" search="walk.all" path="%LocalAppData%\Mozilla\Firefox\Profiles\*\OfflineCache"/>
+    This Linux path is whitelisted under the System - Cache cleaner, so it is not
+    redundant. Also, this Linux path contains the cache2 and OfflineCache directories.
+    -->
+    <action command="delete" search="walk.all" path="$$localprofile$$\cache2"/>
+    <action command="delete" search="walk.all" path="$$localprofile$$\jumpListCache"/>
+    <action command="delete" search="walk.all" path="$$localprofile$$\OfflineCache"/>
+    <action command="delete" search="walk.all" path="$$localprofile$$\startupCache"/>
+    <action command="delete" search="walk.all" path="$$localprofile$$\thumbnails"/>
```

Comment:
These $$localprofile$$ seem to exist only under Windows. If these exists on Windows for  derived browsers (e.g., Pale Moon) then the multi-value variable should stay. Have you seen these for derived browsers?

Reply:
Think they will come to Linux, too! ^^ I'm already using FF68.0a1... ^^ The folders are really new...
Hadn't yet the time to set up VMs and test derived browsers... Why you asked for them ???